packages/eslint-plugin/tests/rules/no-unsafe-return.test.ts

```diff
@@ -408,5 +428,83 @@ function bar() {
         },
       ],
     },
+    {
+      code: `
+declare const value: any;
+async function foo() {
+  return value;
+}
+      `,
+      errors: [
+        {
+          messageId: 'unsafeReturn',
+          line: 4,
+          column: 3,
+          endColumn: 16,
```

Comment:
(nit, optional) If you're gonna include the `endColumn`, might as well include the `endLine` too for consistency. Here and elsewhere